games-strategy/7kaa/7kaa-2.15.4p1.recipe

```diff
@@ -0,0 +1,86 @@
+SUMMARY="Seven Kingdoms: Ancient Adversaries"
+DESCRIPTION="Seven Kingdoms, designed by Trevor Chan, brings a unique blend of Real-Time Strategy\
+	with the addition of trade, diplomacy, and espionage.\
+	\
+	The game enables players to compete against up to six other kingdoms allowing players to conquer \
+	opponents by defeating them in war (with troops or machines), capturing their buildings with spies, \
+	or offering opponents money for their kingdom."
```

Comment:
DESCRIPTION can be at most 100 characters now, a few lines exceed this number.
No tabs in the beginning of  new line. Space at the end of a line before the backslash.
If you want to split the paragraphs, no backslash at the end or in the new line: https://github.com/haikuports/haikuports/wiki/HaikuPorter-Guidelines#description